data/src/generate_phone_summary.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import csv
+import json
+import logging
+import os
+
+from typing import Any, Dict, List
+
+from data.src.codes import LANGUAGES_PATH, PHONES_SUMMARY_PATH
+
+
+def _wiki_name_and_transcription_level(ele: List[str]) -> str:
+    return ele[3] + ele[4]
+
+
+def _handle_wiki_name(
+    language: Dict[str, Any], file_path: str, modifiers: List[str]
+) -> str:
+    name = language["wiktionary_name"]
+    for modifier in modifiers:
+        if modifier in language:
+            key = file_path[
+                file_path.index("_") + 1 : file_path.rindex("_phone")
+            ]
+            if not key:
+                logging.info(
+                    "Failed to isolate key for %r modifier in %r",
+                    modifier,
+                    file_path,
+                )
+                continue
+            values = language[modifier][key]
+            if "|" in values:
+                values = values.replace(" |", ",")
+            name += f" ({values})"
+    return name
+
+
+def main() -> None:
+    with open(LANGUAGES_PATH, "r", encoding="utf-8") as source:
+        languages = json.load(source)
+    readme_list = []
+    languages_summary_list = []
+    path = "../phones"
+    modifiers = ["dialect", "script"]
+
+    for file_path in os.listdir(path):
+        # Filters out README.md.
+        if file_path.endswith(".md"):
+            continue
+        with open(f"{path}/{file_path}", "r", encoding="utf-8") as phone_list:
+            num_of_entries = sum(1 for line in phone_list)
+        iso639_code = file_path[: file_path.index("_")]
+        transcription_level = file_path[
+            file_path.index("phone") : file_path.index(".")
+        ].capitalize()
+        wiki_name = _handle_wiki_name(
+            languages[iso639_code], file_path, modifiers
+        )
+        row = [
+            iso639_code,
+            languages[iso639_code]["iso639_name"],
+            wiki_name,
+            transcription_level,
+            num_of_entries,
+        ]
+        readme_list.append([f"[phone](phones/{file_path})"] + row)
```

Comment:
If we're only outputting a TSV this can just be: `readme_list.append([file_path] + row)`